docs/getting-started/installation.md

```diff
@@ -251,3 +251,18 @@ podAnnotations: {}
 [ecr]: https://gallery.ecr.aws/aquasecurity/trivy
 [registry]: https://github.com/orgs/aquasecurity/packages/container/package/trivy
 [helm]: https://github.com/aquasecurity/trivy/tree/{{ git.tag }}/helm/trivy
+
+
+## pre-commit hook
+
+Add this to your [pre-commit](https://pre-commit.com/) `.pre-commit-config.yaml` config:
+
+```yaml
+repos:
+-   repo: https://github.com/aquasecurity/trivy.git
+    rev: {{ git.tag }}
```

Comment:
I think we can use latest version of a release or revision of the current commit.
What do you think?